library/core/src/intrinsics.rs

```diff
@@ -1901,11 +1901,21 @@ pub unsafe fn copy_nonoverlapping<T>(src: *const T, dst: *mut T, count: usize) {
 /// ```
 /// use std::ptr;
 ///
+/// /// # Safety:
+/// /// * `ptr` must be correctly aligned for its type and non-zero.
+/// /// * `ptr` must be valid for reads of `elts` contiguous objects of type `T`.
```

Comment:
I never heard of "contiguous objects"? Is this perhaps a new term? Also, `s/of/for`.

Reply:
Try ctrl+f 'contiguous' on https://doc.rust-lang.org/std/vec/struct.Vec.html.

Reply:
Yes, `contiguous` is there but I never heard of `contiguous objects`. Try `rg 'contiguous objects'` in rust or https://github.com/rust-lang/rust/search?q=%22contiguous+objects%22